### PR TITLE
Adding entries in document

### PR DESCRIPTION
**Pull request by tuoping**
New file: novel-auxiliary-options.md, application-examples.md
Add the unit and sequence order of virial in use-deepmd-kit.md
Add hardware-platforms in install.md

### CODE REVIEW

File: doc/use-deepmd-kit.md [hunk 1]
```diff
@@ -33,6 +32,7 @@ Time	| ps
 Length	| Å
 Energy	| eV
 Force	| eV/Å
+Virial  | kBar
```
Comment (amcadmus) [inline doc/use-deepmd-kit.md:35]: The unit of virial is eV.

**Review by amcadmus — CHANGES_REQUESTED** [review]

**Comment by njzjz** [comment]
CI build error is related to https://github.com/actions/virtual-environments/issues/2950

File: doc/use-deepmd-kit.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ $ cat force.raw
  6.737  1.554 -5.587 -2.803  0.062  2.222
 -1.968 -0.163  1.020 -0.225 -0.789  0.343
 ```
-This `force.raw` contains 3 frames with each frame having the forces of 2 atoms, thus it has 3 lines and 6 columns. Each line provides all the 3 force components of 2 atoms in 1 frame. The first three numbers are the 3 force components of the first atom, while the second three numbers are the 3 force components of the second atom. The coordinate file `coord.raw` is organized similarly. In `box.raw`, the 9 components of the box vectors should be provided on each line. In `virial.raw`, the 9 components of the virial tensor should be provided on each line. The number of lines of all raw files should be identical.
```
Comment (njzjz) [inline doc/use-deepmd-kit.md:45]: Can you make sure it’s in this order…

File: doc/use-deepmd-kit.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ $ cat force.raw
...
+This `force.raw` contains 3 frames with each frame having the forces of 2 atoms, thus it has 3 lines and 6 columns. Each line provides all the 3 force components of 2 atoms in 1 frame. The first three numbers are the 3 force components of the first atom, while the second three numbers are the 3 force components of the second atom. The coordinate file `coord.raw` is organized similarly. In `box.raw`, the 9 components of the box vectors should be provided on each line. In `virial.raw`, the 9 components of the virial tensor should be provided on each line in the order `XX YY ZZ XY YX YZ ZY XZ ZX`. The number of lines of all raw files should be identical.
```
Comment (njzjz) [inline doc/use-deepmd-kit.md:45]: Can you make sure it’s in this order…

Comment (amcadmus) [reply doc/use-deepmd-kit.md:45]: the order should be xx xy xz yx yy yz zx zy zz

Comment (tuoping) [reply doc/use-deepmd-kit.md:45]: 捕获

Comment (tuoping) [reply doc/use-deepmd-kit.md:45]: 这是dpdata里边的，判断错误，sorry。

File: doc/use-deepmd-kit.md (hunk 1 above)
Comment (tuoping) [reply doc/use-deepmd-kit.md:35]: The virial in VASP OUTCAR is provided with both units, eV and kB, and dpdata took the ones with unit kB.

Comment (tuoping) [reply doc/use-deepmd-kit.md:35]: 捕获

**Comment by tuoping** [comment]
Please fix this bug with compling.

File: doc/use-deepmd-kit.md (hunk 1 above)
Comment (amcadmus) [reply doc/use-deepmd-kit.md:35]: The virial is scaled to eV, see https://github.com/deepmodeling/dpdata/blob/cf4c99bdfdd938176c37f0910d0bd90b45c5dc77/dpdata/system.py#L1151

**Comment by njzjz** [comment]
> Please fix this bug with compling.

fixed in https://github.com/deepmodeling/deepmd-kit/commit/e1119de1cf09fb1365fec29c9690f63d4e4436fb

File: doc/use-deepmd-kit.md (hunk 1 above)
Comment (tuoping) [reply doc/use-deepmd-kit.md:35]: Got it. Thank you.

**Review by amcadmus — APPROVED** [review]

**Pull request merged**